packaging/installer/README.md

```diff
@@ -78,7 +78,7 @@ operating systems.
     os="General Linux with one-line installer (recommended)"
     svg="linux" />
   <InstallBox
-    to="#run-netdata-with-docker"
+    to="/docs/agent/packaging/installer/methods/docker"
```

Comment:
docker is not in [/docs/agent/packaging/installer/methods](https://github.com/netdata/netdata-learn-docusaurus/tree/master/docs/agent/packaging/installer/methods), what i am missing?

Reply:
Whoops, my bad. Should have those fixed now.

Reply:
From what I see docker install method is not fixed

Reply:
ok ok, just noticed that

- docker
   - to: `/docs/agent/packaging/installer/methods/docker`
   - location: **not in** the `/docs/agent/packaging/installer/methods/`


- other methods
   - to: `/docs/agent/packaging/installer/methods/{name}`
   - location: **in** the `/docs/agent/packaging/installer/methods/` (macos, kubernetes, offline, etc.)

=> that is why I decided that docker method wasn't going to work.

Reply:
Yup, you're right. It should have been `/docs/agent/packaging/docker` all along.

Reply:
cool, haha I was about to give up on this, decided to try one more time to explain it 😄